app/src/test/java/com/example/trainingintive/dog_images_feature/domain/usecase/InsertDogImageUrlUseCaseTest.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.trainingintive.dog_images_feature.domain.usecase
+
+import com.example.trainingintive.dog_images_feature.domain.model.DogImageUrl
+import com.example.trainingintive.dog_images_feature.domain.repository.DogImageRepository
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.verify
+import io.reactivex.rxjava3.core.Completable
+import org.junit.Test
+
+class InsertDogImageUrlUseCaseTest {
+
+    val repository: DogImageRepository = mockk() {
+        every { insert(any()) } returns Completable.complete()
```

Comment:
You should not mock `complete` in class, it would be better to mock it test function, so you can leave mock at top level but mocking of returning positive value `every { insert(any()) } returns Completable.complete()` should be in test function. You should stick to this convention because it will be easier later when you will have more complicated tests.